infrastructure/index.ts

```diff
@@ -130,67 +74,6 @@ const originBucketPolicy = new aws.s3.BucketPolicy("origin-bucket-policy", {
     })),
 });
 
-// archiveBucket receives uploaded tarballs of website builds.
-const archiveBucket = new aws.s3.Bucket("archive-bucket");
-
-// archiveHandler processes those uploads.
-const archiveHandler = new awsx.ecs.FargateTaskDefinition("archive-handler", {
-    container: {
-        image: awsx.ecs.Image.fromPath("archive-handler-image", "./"),
-        memory: 4096,
-        cpu: 4,
-    },
-});
-
-// Handle uploads by running the archiveHandler task, which downloads the file, unpacks
-// it, and synchronizes its contents with S3.
-const cluster = awsx.ecs.Cluster.getDefault();
-archiveBucket.onObjectCreated("archive-bucket-subscription", new aws.lambda.CallbackFunction<aws.s3.BucketEvent, void>("archive-bucket-callback", {
-    runtime: "nodejs10.x",
-    policies: awsx.ecs.TaskDefinition.defaultTaskRolePolicyARNs(),
-    callback: async bucketArgs => {
-        if (!bucketArgs.Records) {
-            return;
-        }
-
-        for (const record of bucketArgs.Records) {
-            console.log(`A file was uploaded to the ${archiveBucket.id.get()} bucket: ${record.s3.object.key}`);
-            const source = `s3://${archiveBucket.id.get()}/${record.s3.object.key}`;
-            const dest = `s3://${websiteBucket.id.get()}`;
-
-            console.log(`Running the container task...`);
-            await archiveHandler.run({
-                cluster,
-                overrides: {
-                    containerOverrides: [
-                        {
-                            name: "container",
-
-                            // Pass the S3 URL of the uploaded tarball and destination
-                            // bucket to the container task.
-                            command: [
-                                source,
-                                dest,
-                            ],
-                        },
-                    ],
-                },
-            });
-        }
-    },
-}));
-
-// logsBucket stores the request logs for incoming requests.
-const logsBucket = new aws.s3.Bucket(
-    "requestLogs",
-    {
-        bucket: `${config.targetDomain}-logs`,
-        acl: "private",
-    },
-    {
-        protect: true,
```

Comment:
This resource was manually removed with a `pulumi state delete` in order to allow this change to proceed. The bucket itself is still around, and I've added a README to its contents explaining what it was used for. 

Additional context here: https://pulumi.slack.com/archives/C6Y560ADV/p1596152486079600